bmrg-flow/templates/deployment-service.yaml

```diff
@@ -123,7 +143,13 @@ spec:
       {{- if and (eq $k "controller") (eq $values.workers.logging.type "elastic" ) }}
       - name: secret-volume
         secret:
-          secretName: {{ $values.workers.logging.secret }}
+          secretName: {{ include "bmrg.name" (dict "context" $ "tier" "certs" ) }}
+      {{- else }}
+      {{- if $.Values.database.mongodb.tls.enable }}
```

Comment:
The IF statement should be a single IF statement with an OR

Reply:
As per previous comment, we need to know whether we need to mount secret for one or the other or both.

Reply:
Were you able to combine it? im not seeing the change come through